code/modules/clothing/clothing.dm

```diff
@@ -38,6 +38,9 @@
 	/// Trait modification, lazylist of traits to add/take away, on equipment/drop in the correct slot
 	var/list/clothing_traits
 
+	/// Whether the clothing applies a fov trait, which limits a users field of view.
+	var/fov_trait
```

Comment:
This should be a component or an element, not a var

Reply:
This. It's doable.

Reply:
Yeah I'll get on that in a bit.

Reply:
And it is now!